internal/capability/operator_install_test.go

```diff
@@ -0,0 +1,90 @@
+package capability
+
+import (
+	"context"
+	"os"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/opdev/opcap/internal/operator"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	operatorv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Operator install tests", func() {
+	os.Chdir(os.TempDir())
+
+	csv := operatorv1alpha1.ClusterServiceVersion{
```

Comment:
This is a bit of an anti-pattern. It's ok to _declare_ the variables here, but actual initialization should be done in a setup node.

https://onsi.github.io/ginkgo/#avoid-spec-pollution-dont-initialize-variables-in-container-nodes